index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
minor nit, but if rather than creating the new variables:

```js
var version = null
var versionMsg = null
```

We simply override the variables being passed in:

```
self.version = function (opt, msg, ver) {
  if (arguments.length === 1) {
   version = opt
 } else if (arguments.length === 2) {
   version = msg
 }
 versionOpt = opt || 'version'

usage.version(ver)
self.boolean(versionOpt)
self.describe(versionOpt, msg || usage.deferY18nLookup('Show version number'))
```

We can get things a little bit more concise \o/